README.md

```diff
@@ -6,6 +6,10 @@ Continuous integration server launches Qserv build and also multi-node integrati
 
 [![Build Status](https://travis-ci.org/lsst/qserv.svg?branch=master)](https://travis-ci.org/lsst/qserv)
 
+[![Code Climate](https://codeclimate.com/github/lsst/qserv/badges/gpa.svg)](https://codeclimate.com/github/lsst/qserv)
+
+[![Issue Count](https://codeclimate.com/github/lsst/qserv/badges/issue_count.svg)](https://codeclimate.com/github/lsst/qserv)
```

Comment:
@jhoblitt proposed me to use codeclimate, let's stay its in test for LSST-DM. It'll be easy to remove if LSST does not approve it.